src/useFlags.ts

```diff
@@ -1,5 +1,9 @@
 import { useContext } from 'react';
-import context from './context';
+import context, { LDContext } from './context';
+
+interface LDFlagSet {
```

Comment:
This is a suggestion, but if you export this interface out then end users could use module augmentation to extend out the base LDFlagSet interface. Packages like react-redux use this to help extend out interfaces. This is a snippet from @types/react-redux package with their comment about module augmentation:

```js
/**
 * This interface can be augmented by users to add default types for the root state when
 * using `react-redux`.
 * Use module augmentation to append your own type definition in a your_custom_type.d.ts file.
 * https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation
 */
// tslint:disable-next-line:no-empty-interface
export interface DefaultRootState {}
```

Reply:
I updated to use the LDFlagSet from `launchdarkly-js-client-sdk`.  Originally I tried to import it via `launchdarkly-js-sdk-common` which was causing lint errors.  By using the type from `launchdarkly-js-sdk-common` it can stay consistent and the one from that library is exported.